js/base/functions/string.js

```diff
@@ -11,10 +11,9 @@ module.exports =
 
       // hasFetchOHLCV → has_fetch_ohlcv; parseHTTPResponse → parse_http_response
     , unCamelCase: s => s.match (/^[A-Z0-9_]+$/) ? s : (s.replace (/[a-z0-9][A-Z]/g, x => x[0] + '_' + x[1]).replace(/[A-Z0-9][A-Z0-9][a-z]/g, x => x[0] + '_' + x[1] + x[2]).toLowerCase ())
+    , camelCase: s => s.replace (/_(\w)/g, (x, y) => y.toUpperCase ())
```

Comment:
```
> 'fetch_ohlcv'.replace (/_(\w)/g, (x, y) => y.toUpperCase ())
'fetchOhlcv'
```

Reply:
I'm just using that method to convert between camelCase properties of errors and underscore case. Will change to storing the properties in the underscore case and aliasing the camelCase properties